src/Microsoft.AspNetCore.Mvc.ViewFeatures/Rendering/ViewContext.cs

```diff
@@ -170,6 +173,11 @@ public virtual FormContext FormContext
         public Html5DateRenderingMode Html5DateRenderingMode { get; set; }
 
         /// <summary>
+        /// Information from most recent <see cref="DefaultHtmlGenerator.CreateSanitizedId"/> call.
+        /// </summary>
+        public LastSanitizedId LastSanitizedId { get; }
```

Comment:
Looked like that could result in an `ItemsFeature` allocation per request that wouldn't otherwise happen. But I dived deeper into the allocations and see both `ItemsFeature` and `ItemsDictionary` are allocated regardless. So, using `Items` works for me.